DEVELOPMENT.md

```diff
@@ -0,0 +1,132 @@
+# Development
+
+* [Prerequisites](#prerequisites)
+* [Debug via Blink Developer Tools / node-inspector](#debug-via-blink-developer-tools--node-inspector)
+* [Config file options](#config-file-options)
+* [Feature flags](#feature-flags)
+  * [How to add a new flag](#how-to-add-a-new-flag)
+  * [How to configure feature flags](#how-to-configure-feature-flags)
+* [Run in the cloud](#run-in-the-cloud)
+  * [Initial setup (one time per developer machine)](#initial-setup-one-time-per-developer-machine)
+  * [Create/update AWS EC2 cluster](#createupdate-aws-ec2-cluster)
+* [Dive in development](#dive-in-development)
+
+## Install
+
+You may need Python 2.X (required by node-gyp, only for building).
+
+```sh
+yarn
+```
+
+## Run
+
+To connect to the testnet
+
+```sh
+DEBUG=tendermint,leap-node* node index.js --config=https://testnet-node.leapdao.org
+```
+
+## Run local network
+
+Use [leap-sandox](leapdao/leap-sandbox). It is the easiest way to start local leap network at the moment.
+
+[Alternative guide](https://github.com/parsec-labs/parsec-contracts/wiki/Setting-up-local-development-environment) (may be outdated)
+
+## Debug via Blink Developer Tools / node-inspector
+
+Start node with `--inspect` and port `9999` for example:
+
+`$ node --inspect=9999 /path/to/leap-node <usual args>`
+
+You can connect to the debugger via `chrome://inspect` in the Chrom{ e, ium } browser,
+or with tools like [node-inspector](https://github.com/node-inspector/node-inspector).
+
+Connecting to Remote host using ssh:
+
+`$ ssh -vNL9999:127.0.0.1:9999 <user@remote host>...`
+
+Your local port `9999` forwards now to the remote host on address `127.0.0.1` and port `9999`.
+
+## Config file options
+
+```js
+  "bridgeAddr": "0x7b8342412883f4b34f335d4e1391ec190eb887ca",
+  "operatorAddr": "0x9b83018de826c0343af6e682c24e7c91a421755c",
+  "exitHandlerAddr": "0x0b32eb1aaa9b0804852f4fe1b2e6100edb4533d8",
+  "rootNetwork": "https://rinkeby.infura.io",
+  "rootNetworkId": 4,
+  "network": "leap-testnet-gamma",
+  "networkId": 1341,
+```
+
+* `bridgeAddr` — leap [Bridge](https://github.com/leapdao/leap-contracts) contract address
```

Comment:
@troggy - Link to the contract directly? -- `https://github.com/leapdao/leap-contracts/blob/master/contracts/Bridge.sol`

Recommend doing the same for below contracts as well. Easy to refer.

Reply:
yeah.. these all are good points. I mostly didn't touch Development section at all, just moved out from README